pkg/crcontrollers/csibmnode/controller.go

```diff
@@ -105,6 +114,29 @@ func NewController(nodeSelector string, k8sClient *k8s.KubeClient, logger *logru
 	return c, nil
 }
 
+func (bmc *Controller) enableForNode(nodeName string) {
+	bmc.enabledMu.Lock()
+	bmc.enabledForNode[nodeName] = true
+	bmc.enabledMu.Unlock()
+}
+
+func (bmc *Controller) disableForNode(nodeName string) {
+	bmc.enabledMu.Lock()
+	bmc.enabledForNode[nodeName] = false
```

Comment:
Yes, that makes sense

Reply:
We can use map from sync package however it's method work with `interface{}` type: 
```
func (m *Map) Load(key interface{}) (value interface{}, ok bool)
func (m *Map) Store(key, value interface{})
```
this will force us to check type conversion (for safety) and amount of code won't be reduced. I think own "bicycle" here is more readable and maintainable solution.

Reply:
I mostly worried about encapsulation. If you want to make similar solution you need to have structure with map an mutex inside.